R/projpred.R

```diff
@@ -47,7 +52,8 @@
 #' plot(cv_vs)
 #' }
 get_refmodel.brmsfit <- function(object, newdata = NULL, resp = NULL,
-                                 cvfun = NULL, brms_seed = NULL, ...) {
+                                 cvfun = NULL, dis = NULL, brms_seed = NULL,
```

Comment:
I prefer seed being kind of last. Can we do `latent, dis, brms_seed, ...` order?

Reply:
Sure. Perhaps `dis, latent, brms_seed, ...` since `dis` is a long-standing argument of `init_refmodel()` and `latent` is a new one?

Reply:
okay.

Reply:
Done (commit https://github.com/paul-buerkner/brms/pull/1451/commits/c6268cdc206fa3ef1be82a1582dcaeb453b3b749).